site/en/guides/border-router/mdns-discovery.md

```diff
@@ -1,7 +1,9 @@
 # mDNS Discovery
 
 OTBR registers a `_meshcop._udp` service named `OpenThread BorderRouter`. This
-service is configured in the [OTBR CMakeLists.txt file](https://github.com/openthread/ot-br-posix/blob/main/CMakeLists.txt#L36).
+service is configured in the [OTBR CMakeLists.txt file](https://github.com/openthread/ot-br-posix/blob/main/CMakeLists.txt#L36). The 
+Thread Specification requires Thread Border Routers to publish the `_meshcop._udp` service for the Thread Border Agent service.
+Thread Border Router devices may publish other services that they implement but that is up to each specific device and developer.
```

Comment:
Can you adjust the line breaks so they are less likely to wrap here.  Maybe align it to where the break on line 3 is (after "This")

Reply:
Line breaks added.